client/templates/orderDetails/orderDetails.js

```diff
@@ -126,6 +135,9 @@ Template.orderDetails.helpers({
   },
   hasNonPickableItems: function () {
     const af = this.advancedFulfillment;
+    if (!af.damageCoverage) {
+      return false;
+    }
```

Comment:
@spencern  this was a random bug, when orders had none, so I added a quick exit.